assets/js/src/form_editor/store/blocks_to_form_body.jsx

```diff
@@ -1,5 +1,16 @@
 import { has } from 'lodash';
 
+const mapBlockStyles = (styles) => {
+  const mappedStyles = {
+    full_width: styles.fullWidth ? '1' : '0',
```

Comment:
I think we use '' in the database and not '0'

Reply:
It seems that we use '0' and we check for '1'. I'll keep '0' although I don't like it. It is consistent with other properties.